### PR TITLE
[PPML] Add safe lgbm spark version

### PR DESCRIPTION
**Pull request by hzjane**
## Description
Add secure distributed lgbm spark version.

### CODE REVIEW

File: ppml/trusted-big-data-ml/scala/docker-occlum/Dockerfile [hunk 1]
```diff
@@ -160,6 +160,11 @@ RUN cd /opt && \
     cd /opt/zoo-tutorials/tpch-pyspark && \
     zip -r tpch.zip ./tpch
 
+#LGBM make
+RUN mkdir /ppml && cd /ppml && \
+    git clone --recursive https://github.com/analytics-zoo/LightGBM.git && \
+    cd LightGBM && mkdir build && cd build && \
+    cmake -DUSE_SWIG=ON -DUSE_SSL_SOCKET=1 .. && make -j4
```
Comment (Uxito-Ada) [inline ppml/trusted-big-data-ml/scala/docker-occlum/Dockerfile:167]: Pls check trusted-machine-learning/lgbm/make-lgbm.sh to see how to cmake lgbm

File: ppml/scripts/generate-ssl.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (qiyuangong) [inline ppml/scripts/generate-ssl.sh:1]: Can we use this script for key generation?
https://github.com/intel-analytics/BigDL/blob/main/ppml/scripts/generate-keys.sh

File: ppml/trusted-big-data-ml/scala/docker-occlum/Dockerfile [hunk 3]
```diff
@@ -160,6 +161,11 @@ RUN cd /opt && \
     cd /opt/zoo-tutorials/tpch-pyspark && \
     zip -r tpch.zip ./tpch
 
+#LGBM make
+RUN mkdir /ppml && cd /ppml && \
+    git clone --recursive https://github.com/analytics-zoo/LightGBM.git && \
+    cd LightGBM && mkdir build && cd build && \
+    cmake -DUSE_SWIG=ON -DUSE_SSL=1 .. && make -j4
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/scala/docker-occlum/Dockerfile:168]: Only keep necessary files. Then remove the whole dir

**Review by qiyuangong — APPROVED** [review]
LGTM

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md [hunk 4]
```diff
@@ -134,6 +134,48 @@ Parameters:
 * -d means max_depth: Int.
 We recommend using hdfs to read input-data and write output-result instead of mouting data.
 
+## BigDL LGBM Example
```
Comment (Uxito-Ada) [inline ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md:137]: trusted-big-data-ml has already been deprecated and pls move these to trusted-machine-learning @hzjane

**Pull request merged**